### ISSUE TITLE
FrameManager.NavigateFrame timeout error

### ISSUE THREAD

**Issue by inancgumus** [grafana/xk6-browser#76]
```
$ go test ./tests ./common -race -v -count=100 -failfast -run="TestPageGoto"

=== RUN   TestPageGoto/Page.goto/should_work
time="2021-11-09T12:15:18+03:00" level=warning msg="error fetching response body: No data found for resource with given identifier (-32000)" category=cdp elapsed="0 ms" goroutine=10921
--- FAIL: TestPageGoto (30.88s)
    --- FAIL: TestPageGoto/Page.goto (30.55s)
        --- FAIL: TestPageGoto/Page.goto/should_work (30.55s)
panic: timed out [recovered]
        panic: timed out

goroutine 10915 [running]:
testing.tRunner.func1.2({0x2346020, 0xc001fff3b0})
        /usr/local/Cellar/go/1.17.1/libexec/src/testing/testing.go:1209 +0x36c
testing.tRunner.func1()
        /usr/local/Cellar/go/1.17.1/libexec/src/testing/testing.go:1212 +0x3b6
panic({0x2346020, 0xc001fff3b0})
        /usr/local/Cellar/go/1.17.1/libexec/src/runtime/panic.go:1047 +0x266
go.k6.io/k6/js/common.Throw(...)
        /Users/inanc/go/pkg/mod/go.k6.io/k6@v0.34.2-0.20211105090803-dae70006ca77/js/common/util.go:36
github.com/grafana/xk6-browser/common.(*FrameManager).NavigateFrame(0xc000c04480, 0xc00259a680, {0xc0005e06c0, 0x21}, {0x0, 0x0})
        /Users/inanc/grafana/xk6-browser/common/frame_manager.go:441 +0xf05
github.com/grafana/xk6-browser/common.(*Frame).Goto(0xc00259a680, {0xc0005e06c0, 0x21}, {0x0, 0x0})
        /Users/inanc/grafana/xk6-browser/common/frame.go:730 +0x6d
github.com/grafana/xk6-browser/common.(*Page).Goto(0xc00202c000, {0xc0005e06c0, 0x21}, {0x0, 0x0})
        /Users/inanc/grafana/xk6-browser/common/page.go:428 +0x8c
github.com/grafana/xk6-browser/tests.testPageGoto(0x0, 0xc0026bcdc0)
        /Users/inanc/grafana/xk6-browser/tests/page_goto_test.go:50 +0x1a3
github.com/grafana/xk6-browser/tests.TestPageGoto.func1.1(0x0)
        /Users/inanc/grafana/xk6-browser/tests/page_goto_test.go:38 +0x3b
testing.tRunner(0xc000d5c340, 0xc00242d680)
        /usr/local/Cellar/go/1.17.1/libexec/src/testing/testing.go:1259 +0x230
created by testing.(*T).Run
        /usr/local/Cellar/go/1.17.1/libexec/src/testing/testing.go:1306 +0x727
FAIL    github.com/grafana/xk6-browser/tests    56.724s
testing: warning: no tests to run
PASS
ok      github.com/grafana/xk6-browser/common   0.087s [no tests to run]
FAIL
```

**Comment by inancgumus** [comment]
I can't reproduce this locally :(

I tried to install `perflock` to minimize system resources, also tried `GOMAXPROCS=2`, but they didn't help.

**Comment by imiric** [comment]
This should be fixed by #480.
